app/i18n/locales/en.js

```diff
@@ -704,5 +704,7 @@ export default {
 	The_user_will_be_removed_from_s: 'The user will be removed from {{s}}',
 	Yes_remove_user: 'Yes, remove user!',
 	Direct_message: 'Direct message',
-	Message_Ignored: 'Message ignored. Tap to display it.'
+	Message_Ignored: 'Message ignored. Tap to display it.',
+	Enter_workspace_URL: 'Enter workspace URL',
+	Workspace_URL_Example: 'Ex. your-company.rocket.chat'
```

Comment:
If I provide key-value 'Ex. your-company.rocket.chat', will result in an error.
For that reason, changed it's key name to be Workspace_URL_Example

<img width="301" alt="Screenshot 2020-12-07 at 5 33 02 PM" src="https://user-images.githubusercontent.com/23723464/101349048-b233b780-38b2-11eb-9a1b-0c8acd9b44ee.png">